projects/plugins/jetpack/changelog/add-sig-toggle-jetpack-settings

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: major
```

Comment:
I don't know that I would classify this as a major enhancement. I think enhancement would be enough.

```suggestion
Type: enhancement
```